src/test/java/de/tonypsilon/bmm/backend/division/service/DivisionServiceTest.java

```diff
@@ -190,24 +196,23 @@ void testCreateDivisionInvalidLevel() {
 
         BadDataException actualException = assertThrows(BadDataException.class,
                 () -> divisionService.createDivision(divisionCreationData));
-        assertEquals("Das Level der Staffel muss eine ganze Zahl > 0 sein!",
-                actualException.getMessage());
+        assertThat(actualException.getMessage()).isEqualTo("Das Level der Staffel muss eine ganze Zahl > 0 sein!");
     }
 
     @Test
     void testDivisionExistsById() {
         when(divisionRepository.existsById(1L)).thenReturn(Boolean.TRUE);
         when(divisionRepository.existsById(2L)).thenReturn(Boolean.FALSE);
-        assertEquals(Boolean.TRUE, divisionService.divisionExistsById(1L));
-        assertEquals(Boolean.FALSE, divisionService.divisionExistsById(2L));
+        assertThat(divisionService.divisionExistsById(1L)).isEqualTo(Boolean.TRUE);
```

Comment:
AssertJ offers .isTrue and .isFalse for assertions on Booleans. They can be used here.